src/src/rx_main.cpp

```diff
@@ -928,6 +932,10 @@ static void setupGpio()
     pinMode(GPIO_PIN_UART1TX_INVERT, OUTPUT);
     digitalWrite(GPIO_PIN_UART1TX_INVERT, LOW);
 #endif
+
+#if defined(GPIO_PIN_SDA) && GPIO_PIN_SDA != UNDEF_PIN
```

Comment:
`GPIO_PIN_SCL` pin should be checked as well. 
Same for tx as well.... maybe it would be good to add a `commonSetupTarget()` method.

Reply:
Good idea, I've made a new `setupTargetCommon()` for both RX and TX and converted the checks to include both SDA and SCL. 

Sorry about the delay in addressing this, I knew there was one PR somewhere where someone had made a good point but I saw it while reading my phone in bed and could not remember which PR it was on!

Reply:
Thanks, I still think that function ahould be in `common.cpp` which will make common for both RX and TX (same code in 2 places which can be merged and will make maintaining easier).
Otherwise `common` is a bit misleading naming...

Reply:
Oh you didn't say common.cpp! I thought you meant a setup that is common to all targets and not setting up target-specific GPIOs. I'm not sure it belongs in common.cpp though, since there's no platform code at all in there. common.cpp really is all OTA stuff really and doesn't deal with peripherals so I'd disagree that it belongs there.

If we were going to make a `commonSetup()` then there's a lot of code that might be as candidate for consolidation. Doing that opens up the possibility of a lot of weird problems as the TX and RX setup is merged, and that's getting outside the scope of this PR, which was literally just moving one line of code to execute at a different place and became header sanity and proper Wire initialization (instead of multiple devices using it and the order they're in an array was significant).

So given that, should I move `Wire.begin` back to each app's `setupTarget()` then?

Reply:
You have a point and `common.cpp` should be renamed then to more descriptive.
Maybe a new `platform.cpp` (or something) should be created to maintain common RX and TX codes + own files for MCUs to eliminate need for huge amount of `ifdef`, `elif`, `else` pairs...
Anyway, that particular I2C init code is identical for RX and TX so there wont be any issue to make it generic (just a single function call). Then other common code can be moved piece by piece as well later. No point to keep same code in multiple places, imo. Or do you think it makes more sense? 🙃
RX and TX specific code should be kept separated as it is today...

Reply:
common should definitely be renamed or moved. It is all OTA stuff and we already have an OTA unit so it should be moved to that lib for sure. (and for chrissake someone rename `ExpressLRS_currAirRate_Modparams` and `ExpressLRS_currAirRate_RFperfParams`!

But not today! I made a new unit rxtx_common.cpp with this one line of common code as we discussed. So now both rx and tx have a line which calls the function that has one line. But now it exists for any other future common target setup.